lib/result-view.js

```diff
@@ -113,6 +113,8 @@ export default class ResultView {
       return;
     }
 
+    console.log('ResultView: Add result', result);
+
```

Comment:
I only moved it a few lines down in order to remove the logs generated from execution_count and status events.